### DISCUSSION TITLE
Auth callback on server-side

### DISCUSSION THREAD

**Discussion by tessamerrill** [supabase/supabase#6707] [Questions]
Hi there! I was wondering if its at all possible to have the authentication callback on the server-side, such as an API.

TL;DR my issue is that I'm using [supabase-go](https://github.com/nedpals/supabase-go), and I can't access the access_token sent by GoTrue on the auth callback (since it's in a URL fragment, the server can't access that). 

I've already submitted [an issue](https://github.com/supabase/gotrue/issues/472) to GoTrue hoping that this can be changed.

I'm trying to avoid using client-side Javascript so my users can *technically* have JS disabled, and still use my application (like GitHub).

Thanks!

**Comment by GaryAustin1** [comment]
This thread, with the SB devs, seems to be discussing your topic, although it does not appear solved yet...
https://github.com/supabase/gotrue/issues/279

**Reply by tessamerrill** [reply]
> This thread, with the SB devs, seems to be discussing your topic, although it does not appear solved yet...
> 
> https://github.com/supabase/gotrue/issues/279

Ohh okay, thanks! I saw that but didn't think much of it, so I'll keep an eye out on the issue, might try to contribute too if I can.
